XCUITests/TopTabsTest.swift

```diff
@@ -184,24 +184,6 @@ class TopTabsTest: BaseTestCase {
         navigator.performAction(Action.AcceptRemovingAllTabs)
         XCTAssertTrue(app.staticTexts["Private Browsing"].exists, "Private welcome screen is not shown")
     }
-    // This test is disabled, this option is not shown now
-    func testCloseTabFromPageOptionsMenu() {
-        // Open two websites so that there are two tabs open and the page options menu is available
-        navigator.openURL(urlValue)
-        navigator.openNewURL(urlString: urlExample)
-        checkNumberOfTabsExpectedToBeOpen(expectedNumberOfTabsOpen: 2)
-
-        // Go back to one website so that the page options menu is available and close one tab from there
-        closeTabTrayView(goBackToBrowserTab: urlLabelExample)
-        navigator.performAction(Action.CloseTabFromPageOptions)
```

Comment:
This action could be removed from the FxScreenGraph and also if there is an screen or more code related